card/lib/card/subcards.rb

```diff
@@ -236,11 +236,20 @@ def extract_subcard_args! name, args
       subcards
     end
 
+    # ensure a leading '+'
+    def normalize_subfield_key key
+      if key.is_a?(Symbol) && Card::Codename.exist?(key)
+        key = Card::Codename.name(key)
+      end
+      key = "+#{key}" unless key.to_s.start_with? "+"
+      key
+    end
+
     def new_by_attributes name, attributes={}
       attributes ||= {}
       absolute_name = absolutize_subcard_name name
       if absolute_name.field_of?(@context_card.name) &&
-         (absolute_name.parts.size - @context_card.name.parts.size) > 2
+         (absolute_name.parts.size - @context_card.name.parts.size) >= 2
```

Comment:
I'm pretty sure this is never going to pass.  If `absolute_name.field_of?(@context_card.name)` is true, then the result of this subtraction will ALWAYS be exactly 1.